cmd/builder/test/test.sh

```diff
@@ -69,28 +66,29 @@ test_build_config() {
         # Since the content of the servicez page depend on which extensions are
         # built into the collector, we depend only on the zpages extension
         # being present and serving something.
-        curl --fail --silent --output /dev/null http://localhost:55679/debug/servicez
-        if [ $? == 0 ]; then
+        if curl --fail --silent --output /dev/null http://localhost:55679/debug/servicez; then
```

Comment:
Should this be `if !`?

Reply:
I think it shouldn't be `if !`
From https://www.shellcheck.net/wiki/SC2181:
"To check that a command returns success, use `if mycommand; then ...` ."

Reply:
Ah.... i see according to the link you posted that `!` for command failure is the opposite of what I expected. Thanks for the info